update disclaimer wording (#1554)
agrc/gis.utah.gov: about/policy/license-disclaimer/index.md

## Disclaimer
{: .text-left}

> These data, including (but not limited to) geographic data, tabular data, or analytical data, are provided "as is" without any express or implied warranties.

Author: @jacobdadams should "express" be "expressed or implied warranties"?

Reviewer: I think "express" is pretty common verbiage in this scenario.

Reviewer 2: I think it's "express". If I understand the UCC link correctly, there are "express" warranties and "implied" warranties—"express" and "implied" are adjectives, not verbs, so we don't change the tense.

> The data herein, including but not limited to geographic data, tabular data, analytical data, electronic data structures or files, are provided as is; without warranty of any kind, either expressed or implied, or statutory, including, but not limited to, the implied warranties or merchantability and fitness for a particular purpose. 

Reviewer 2: Unless this is coming from a lawyer or a higher-level disclaimer, I think we're better served by simplifying the language. Digging into different warranties yielded this link to the Uniform Commercial Code, which suggests "as is" is sufficient to disclaim all implied warranties without listing them all: https://www.law.cornell.edu/ucc/2/2-316
```suggestion
These data, including (but not limited to) geographic data, tabular data, or analytical data, are provided "as is" without any express or implied warranties.
```

> This data is provided as a public service and should be used for informational purposes only. AGRC reserves the right to change or revise published data and/or these services at any time. 

Reviewer 2: I feel the need to include a statement here about GIS=get it surveyed. 

```suggestion
These data are provided as a public service and should be used for informational purposes only. They cannot replace the proper evaluation of a location by a qualified professional. AGRC reserves the right to change or revise published data and/or services at any time. 
```

Author: good call.

> AGRC and the state of Utah is not responsible for the misuse or misrepresentation of the data. The entire risk as to the quality and performance of the data is assumed by the user.

Reviewer 2: My inner crusade against passive voice is coming out.

```suggestion
Neither AGRC nor the State of Utah are responsible for the misuse or misrepresentation of the data. The user assumes the entire risk as to the quality and performance of the data.
```

Author: i need an inner crusader like that.  yup, that sounds better.  thanks!
